src/Application.php

```diff
@@ -569,4 +566,20 @@ public function getVersion($long = true)
 
         return $this['bolt_version'];
     }
+
+    /**
+     * Generates a path from the given parameters.
+     *
+     * Note: This can be pulled in from Silex\Application\UrlGeneratorTrait
+     * once we support Traits.
+     *
+     * @param string $route      The name of the route
+     * @param array  $parameters An array of parameters
+     *
+     * @return string The generated path
+     */
+    public function path($route, $parameters = array())
+    {
+        return $this['url_generator']->generate($route, $parameters);
+    }
```

Comment:
This is the same as what `UrlGeneratorTrait` and `Library` call. But yes I agree. 
I thought about changing it to `generatePath`